core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -1386,4 +1390,23 @@ export class WorkflowEditorComponent implements AfterViewInit, OnDestroy {
         }
       });
   }
+
+  /**
+   * Handles mouse events to enable shared cursor.
+   */
+  private handlePointerEvents(): void {
+    // eslint-disable-next-line rxjs-angular/prefer-takeuntil
```

Comment:
do we really need this disable? I try to delete it and no warnings show up

Reply:
My IDE has ESLint errors. It depends on ESLint settings. But it is best to add `untilDestroyed` in places like this so I added it and resolved the ESLint errors.